test/unit/CMakeLists.txt

```diff
@@ -0,0 +1,58 @@
+cmake_minimum_required(VERSION 3.14...3.22)
+
+project(mallocMCUnitTests LANGUAGES CXX)
+
+# ---- Options ----
+
+option(ENABLE_TEST_COVERAGE "Enable test coverage" OFF)
```

Comment:
Those variables should start with `mallocMC_` to clearly show the users that mallocMC behaviour is changed.
This will also group all variables of mallocMC which helps finding all possible cmake switches.